com.sap.cloud.lm.sl.cf.client/src/main/java/com/sap/cloud/lm/sl/cf/client/ResilientCloudControllerClient.java

```diff
@@ -1027,4 +1039,10 @@ public void unbindService(String applicationName, String serviceName,
         executeWithRetry(() -> cc.unbindService(applicationName, serviceName, applicationServicesUpdateCallback));
     }
 
+	@Override
+    public CloudResources getKnownRemoteResources(CloudResources applicationResources) {
+        return executeWithRetry(() -> {
+            return cc.getKnownRemoteResources(applicationResources);

```

Comment:
Why do we need this?
Could it be `return executeWithRetry(() -> cc.getKnownRemoteResources(applicationResources)); `?